Dockerfile

```diff
@@ -56,8 +57,14 @@ RUN make -C /go/src/github.com/envoyproxy/protoc-gen-validate/ build
 
 RUN go get -u github.com/mwitkow/go-proto-validators/protoc-gen-govalidators
 
-# Add grpc-web support
 
+
+# Add scala support
```

Comment:
Do we want to make this one `RUN` command? I saw both multiple `RUN` commands as well as `&&` commands in the `Dockerfile`.

Reply:
Yeah if we could collapse this to a single RUN it will generate less layers